app/assets/styles/locations/_single.scss

```diff
@@ -148,8 +138,33 @@
   }
 }
 
-/* Fold: Dataset
-   ========================================================================== */
+.source__title {
+  color: $primary-color;
```

Comment:
Not sure, but I'd think this should only be blue if clickable? Should the coloring move to `--external`?

Reply:
Thanks to @rwegener2's location shared above I understood that the source name is never clickable, there will be an extra field displayed. (Still, I find it confusing that a non-clickable element has the same color as a link usually has)

Reply:
According to the screenshot on the issue, the differentiation lies in the icon. @tania-pires is this correct?

Reply:
I agree the coloring is a little confusing.  Seems like this would be more than a minor fix, though?  We would either be changing the color of all the light blue text on the website (or at least the dashboard page) or all the links.